stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -182,6 +182,53 @@ for elty in (Float32,Float64)
     end
 end
 
+"""
+    muladd(A, y, z)
+
+Combined multiply-add, `A*y .+ z`, for matrix-matrix or matrix-vector multiplication.
+The result is always the size of `A*y`, but `z` may be smaller, or a scalar.
```

Comment:
I'm not a native speaker, but shouldn't it be
```suggestion
The result is always of the size of `A*y`, but `z` may be smaller, or a scalar.
```
? Otherwise, one may expect a tuple of `Int`s.

Reply:
Thanks I didn't spot that reading. How about "The result is always the same size as `A*y`,"?